tfidf-benchmark/NLP-pipeline-cuML.ipynb

```diff
@@ -0,0 +1,4769 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        vectorizer = HashingVectorizer(stop_words='english', preprocessor=None)
</pre><p>You are replacing stop_words twice. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>